### PR TITLE
Pic 3271 - Update what's new

### PR DESCRIPTION
**Pull request by johnyfarrar**
Update What's new with https://dsdmoj.atlassian.net/jira/software/c/projects/PIC/boards/980?modal=detail&selectedIssue=PIC-3271&assignee=63fe35670a4a47fb8d213f5d

### CODE REVIEW

File: server/views/whats-new.njk [hunk 1]
```diff
@@ -20,6 +20,25 @@
...
+            <ul class="govuk-list govuk-list--bullet">
+                <li>source - so you can just see Common Platform cases or just Libra cases</li>
+                <li>flag - so you can see only breach cases</li>
+            </ul>
+
+            <p class="govuk-body">We hope this will help you find the cases you need more quickly.</p>
+
+            <h3 class="govuk-heading-s">Speed improvements</h3>
+
+            <p class="govuk-body">We have made some changes in the background of the service so that pages will load more quickly. If you have seen an error that says 'There is a problem with the service, try again later' recently, one of the reasons why is because the pages are loading too slowly. </p>
```
Comment (rschallapalli) [inline server/views/whats-new.njk:38]: Should we be specific that the improvements are specific to case list page?

**Review by rschallapalli — CHANGES_REQUESTED** [review]

File: server/views/whats-new.njk (hunk 1 above)
Comment (johnyfarrar) [reply server/views/whats-new.njk:38]: I think that's a discussion for the ticket and/or stand up? But agree you are right there with the terminology

Comment (johnyfarrar) [reply server/views/whats-new.njk:38]: Updated from slack content

**Review by rschallapalli — APPROVED** [review]
LGTM

**Pull request merged**